app/assets/stylesheets/base/mixins.scss

```diff
@@ -603,25 +645,25 @@ ORDER: Base + typography > general layout + grid > page layout > components
   ul li {
     display: flex;
     justify-content: space-between;
-    border-bottom: 1px solid v(cu-light-grey);
+    @include v(border-bottom, solid-grey);
 
```

Comment:
Another example of how the mixin works and the css variable it creates using the css property combined with the Sass variable

Reply:
<img width="346" alt="Screenshot 2020-05-11 15 31 39" src="https://user-images.githubusercontent.com/29960336/81618594-8728ca00-939c-11ea-9ca4-c33a3511edcb.png">

Reply:
This is dope; I have two concerns:

1) If we don't leave the current `var(--cu-red)` in place, we can't use them in the browser inspector, which I use often to eyeball colors

<img width="1149" alt="Screen Shot 2020-05-11 at 3 38 43 PM" src="https://user-images.githubusercontent.com/8109093/81619132-bab82400-939d-11ea-8659-81f2d2435dcc.png">

2) Dependent on 1) (if we don't leave the vars in place), we should do a search all and replace the few straggler CSS vars using that syntax. Did that [here](https://github.com/chapmanu/cascade-assets/tree/ie-fix__nick) if you want to merge.

I'd really prefer leaving the CSS vars in 1) if you all are cool with it. Maybe we should move all the colors back to their own SCSS file for organization

Reply:
Lastly, would it be less work to just use a polyfilly since we're only worried about IE?
https://github.com/nuxodin/ie11CustomProperties 
https://github.com/polyfills/variables